libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/bulletChart/tests/PluggableBulletChart.test.tsx

```diff
@@ -482,4 +492,32 @@ describe("PluggableBulletChart", () => {
             });
         });
     });
+
+    describe("Drill Down", () => {
+        it.each([
+            [sourceInsightDef, Region, targetUri, intersection, expectedInsightDefRegion],
+            [sourceInsightDef, Department, targetUri, intersection, expectedInsightDefDepartment],
+        ])(
+            "should replace the drill down attribute and add intersection filters",
```

Comment:
The test name does not reflect the input.

Reply:
Added test annotation